org.remmina.Remmina.json

```diff
@@ -42,9 +42,7 @@
         "--talk-name=org.gnome.Shell",
         /* Local network host discovery */
         "--system-talk-name=org.freedesktop.Avahi",
-        /* Needed for dconf to work */
-        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
-        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+        "--metadata=X-DConf=migrate-path=/org/remmina/Remmina/"
```

Comment:
I saw you blindly add this to another PR. Does this actually use settings at this path?

Reply:
now that I checked with dconf editor, there seems to be no traces of remmina anywhere. Are these permissions (and the metadata line) unnecessary then?